docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -1316,9 +1469,13 @@ Number of compatible differences between published cluster states.
     (integer)
     Number of documents transformed by the processor.
 
+`ingest.pipelines.<pipeline_id>.<processor>.time`::
```

Comment:
This should be `ingest.pipelines.<pipeline_id>.processors.<processor>.time` since the processor object is nested under an array field. This should probably be updated for all of the processor specific stats in this reference, though that might be fine to do in a separate PR.

Reply:
Good catch. Fixed this up with 28f5be11954e88bc19baf66387ab0066a8b22459.